### DISCUSSION TITLE
Page 36 | Chapter 2 向量运算 | Book 4《矩阵力量》

### DISCUSSION THREAD

**Discussion by zongyilileo** [Visualize-ML/Book4_Power-of-Matrix#128] [General]
姜老师您好：
最近在学习您的这套教程。
偶然发现pdf文件36页公式93第一行最后一项系数，是否应为 b_n？不知您是否已经发现这处typo。
顺便向您致敬，这套教程设计的太好了，相见恨晚

**Comment by Visualize-ML** [comment]
谢谢你的肯定！这处还真没错，b_m，强调a和b向量长度不同。可能需要注释一下。

也请继续批评指正。

Cheers,
James

**Reply by zongyilileo** [reply]
目前在看您github上的版本。此处还是b_1。可能是版本的问题

Regards,
Leo

**Comment by Visualize-ML** [comment]
谢谢你，很多错误都在纸质书上修改了。但是，电子草稿没有同步。

也请你继续批评指正。

Cheers,
James
